--- origsrc/kdebase-runtime-4.1.4/nepomuk/server/CMakeLists.txt	2008-05-29 06:18:48.000000000 -0500
+++ src/kdebase-runtime-4.1.4/nepomuk/server/CMakeLists.txt	2009-01-25 14:28:05.000500000 -0600
@@ -58,7 +58,7 @@ install(
 
 install(
   TARGETS kdeinit_nepomukserver
-  DESTINATION ${LIB_INSTALL_DIR})
+  DESTINATION ${INSTALL_TARGETS_DEFAULT_ARGS})
 
 install(
   TARGETS nepomukserver
--- origsrc/kde-runtime-4.7.0/nepomuk/services/storage/lib/CMakeLists.txt	2011-07-07 09:03:18.000000000 -0500
+++ src/kde-runtime-4.7.0/nepomuk/services/storage/lib/CMakeLists.txt	2011-08-08 12:14:18.870773900 -0500
@@ -40,7 +40,7 @@ target_link_libraries(nepomukdatamanagem
   ${NEPOMUK_LIBRARIES}
   ${KDE4_KDECORE_LIBS})
 
-install(TARGETS nepomukdatamanagement DESTINATION ${LIB_INSTALL_DIR})
+install(TARGETS nepomukdatamanagement ${INSTALL_TARGETS_DEFAULT_ARGS})
 install(FILES
   simpleresource.h
   simpleresourcegraph.h
