--- origsrc/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/dbusoperators.h	2008-09-26 09:55:58.000000000 -0500
+++ src/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/dbusoperators.h	2009-01-28 19:02:59.609875000 -0600
@@ -29,20 +29,20 @@
 
 namespace Nepomuk {
     namespace Search {
-        NEPOMUK_EXPORT void registerDBusTypes();
+        KDE_EXPORT void registerDBusTypes();
     }
 }
 
-NEPOMUK_EXPORT QDBusArgument& operator<<( QDBusArgument& arg, const Soprano::Node& );
-NEPOMUK_EXPORT const QDBusArgument& operator>>( const QDBusArgument& arg, Soprano::Node& );
+KDE_EXPORT QDBusArgument& operator<<( QDBusArgument& arg, const Soprano::Node& );
+KDE_EXPORT const QDBusArgument& operator>>( const QDBusArgument& arg, Soprano::Node& );
 
-NEPOMUK_EXPORT QDBusArgument& operator<<( QDBusArgument& arg, const Nepomuk::Search::Term& term );
-NEPOMUK_EXPORT const QDBusArgument& operator>>( const QDBusArgument& arg, Nepomuk::Search::Term& );
+KDE_EXPORT QDBusArgument& operator<<( QDBusArgument& arg, const Nepomuk::Search::Term& term );
+KDE_EXPORT const QDBusArgument& operator>>( const QDBusArgument& arg, Nepomuk::Search::Term& );
 
-NEPOMUK_EXPORT QDBusArgument& operator<<( QDBusArgument& arg, const Nepomuk::Search::Query& query );
-NEPOMUK_EXPORT const QDBusArgument& operator>>( const QDBusArgument& arg, Nepomuk::Search::Query& );
+KDE_EXPORT QDBusArgument& operator<<( QDBusArgument& arg, const Nepomuk::Search::Query& query );
+KDE_EXPORT const QDBusArgument& operator>>( const QDBusArgument& arg, Nepomuk::Search::Query& );
 
-NEPOMUK_EXPORT QDBusArgument& operator<<( QDBusArgument& arg, const Nepomuk::Search::Result& );
-NEPOMUK_EXPORT const QDBusArgument& operator>>( const QDBusArgument& arg, Nepomuk::Search::Result& );
+KDE_EXPORT QDBusArgument& operator<<( QDBusArgument& arg, const Nepomuk::Search::Result& );
+KDE_EXPORT const QDBusArgument& operator>>( const QDBusArgument& arg, Nepomuk::Search::Result& );
 
 #endif
--- origsrc/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/query.h	2008-09-26 09:55:58.000000000 -0500
+++ src/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/query.h	2009-01-28 19:02:59.625500000 -0600
@@ -44,7 +44,7 @@ namespace Nepomuk {
          *
          * \author Sebastian Trueg <trueg@kde.org>
          */
-        class NEPOMUK_EXPORT Query
+        class KDE_EXPORT Query
         {
         public:
             enum Type {
@@ -113,10 +113,10 @@ namespace Nepomuk {
             QSharedDataPointer<Private> d;
         };
 
-        NEPOMUK_EXPORT uint qHash( const Nepomuk::Search::Query& );
+        KDE_EXPORT uint qHash( const Nepomuk::Search::Query& );
     }
 }
 
-NEPOMUK_EXPORT QDebug operator<<( QDebug, const Nepomuk::Search::Query& );
+KDE_EXPORT QDebug operator<<( QDebug, const Nepomuk::Search::Query& );
 
 #endif
--- origsrc/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/queryparser.h	2008-09-26 09:55:58.000000000 -0500
+++ src/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/queryparser.h	2009-01-28 19:02:59.641125000 -0600
@@ -29,7 +29,7 @@
 
 namespace Nepomuk {
     namespace Search {
-        class NEPOMUK_EXPORT QueryParser
+        class KDE_EXPORT QueryParser
         {
         public:
             QueryParser();
--- origsrc/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/result.h	2008-09-26 09:55:58.000000000 -0500
+++ src/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/result.h	2009-01-28 19:02:59.656750000 -0600
@@ -38,7 +38,7 @@ namespace Nepomuk {
          *
          * \author Sebastian Trueg <trueg@kde.org>
          */
-        class NEPOMUK_EXPORT Result
+        class KDE_EXPORT Result
         {
         public:
             Result();
--- origsrc/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/term.h	2008-10-30 09:13:02.000000000 -0500
+++ src/kdebase-runtime-4.2.0/nepomuk/libnepomukquery/term.h	2009-01-28 19:02:59.656750000 -0600
@@ -41,7 +41,7 @@ namespace Nepomuk {
          *
          * \author Sebastian Trueg <trueg@kde.org>
          */
-        class NEPOMUK_EXPORT Term
+        class KDE_EXPORT Term
         {
         public:
             /**
@@ -272,10 +272,10 @@ namespace Nepomuk {
             QSharedDataPointer<Private> d; 
         };
 
-        NEPOMUK_EXPORT uint qHash( const Nepomuk::Search::Term& );
+        KDE_EXPORT uint qHash( const Nepomuk::Search::Term& );
     }
 }
 
-NEPOMUK_EXPORT QDebug operator<<( QDebug, const Nepomuk::Search::Term& );
+KDE_EXPORT QDebug operator<<( QDebug, const Nepomuk::Search::Term& );
 
 #endif
--- origsrc/kdebase-runtime-4.2.0/nepomuk/server/main.cpp	2008-09-18 07:25:44.000000000 -0500
+++ src/kdebase-runtime-4.2.0/nepomuk/server/main.cpp	2009-01-28 18:53:53.906750000 -0600
@@ -83,7 +83,7 @@ namespace Nepomuk {
 }
 
 
-extern "C" NEPOMUK_SERVER_EXPORT int kdemain( int argc, char** argv )
+extern "C" KDE_EXPORT int kdemain( int argc, char** argv )
 {
     KAboutData aboutData( "NepomukServer", "nepomuk",
                           ki18n("Nepomuk Server"),
--- origsrc/kdebase-runtime-4.2.0/nepomuk/server/nepomukserver_export.h	2008-09-18 07:25:44.000000000 -0500
+++ src/kdebase-runtime-4.2.0/nepomuk/server/nepomukserver_export.h	2009-01-28 18:50:09.203625000 -0600
@@ -24,7 +24,13 @@
 #include <kdemacros.h>
 
 #ifndef NEPOMUK_SERVER_EXPORT
-#define NEPOMUK_SERVER_EXPORT KDE_EXPORT
+# if defined(MAKE_NEPOMUK_SERVER_LIB)
+   /* We are building this library */
+#  define NEPOMUK_SERVER_EXPORT KDE_EXPORT
+# else
+   /* We are using this library */
+#  define NEPOMUK_SERVER_EXPORT KDE_IMPORT
+# endif
 #endif
 
 # ifndef NEPOMUK_SERVER_EXPORT_DEPRECATED
--- origsrc/kdebase-runtime-4.2.0/nepomuk/services/queryservice/searchcore.h	2008-10-30 09:13:02.000000000 -0500
+++ src/kdebase-runtime-4.2.0/nepomuk/services/queryservice/searchcore.h	2009-01-28 18:58:09.391125000 -0600
@@ -36,7 +36,7 @@ namespace Nepomuk {
         /**
          * Please consider using QueryServiceClient instead.
          */
-        class NEPOMUK_EXPORT SearchCore : public QObject
+        class KDE_EXPORT SearchCore : public QObject
         {
             Q_OBJECT
 
--- origsrc/kdebase-runtime-4.2.0/nepomuk/services/strigi/priority.cpp	2008-09-18 07:25:43.000000000 -0500
+++ src/kdebase-runtime-4.2.0/nepomuk/services/strigi/priority.cpp	2009-01-28 17:32:00.109875000 -0600
@@ -31,7 +31,9 @@
 #include <sys/resource.h>
 
 #include <unistd.h>
+#ifndef __CYGWIN__
 #include <sys/syscall.h>
+#endif
 #include <errno.h>
 
 #include <sched.h>
