--- origsrc/kdebase-runtime-4.4.2/kioslave/CMakeLists.txt	2009-11-30 18:29:30.000000000 -0600
+++ src/kdebase-runtime-4.4.2/kioslave/CMakeLists.txt	2010-04-28 22:56:05.255228300 -0500
@@ -35,7 +35,9 @@ if(NOT WIN32)
    add_subdirectory( floppy )
    add_subdirectory( finger )
    add_subdirectory( man )
+if(NOT CYGWIN)
    add_subdirectory( nfs )  
+endif(NOT CYGWIN)
 endif(NOT WIN32)
 
 if(SAMBA_FOUND OR WIN32)
