--- origsrc/kdebase-runtime-4.6.4/plasma/declarativeimports/core/CMakeLists.txt	2011-02-25 16:07:08.000000000 -0600
+++ src/kdebase-runtime-4.6.4/plasma/declarativeimports/core/CMakeLists.txt	2011-06-14 19:54:42.532914200 -0500
@@ -19,7 +19,7 @@ INCLUDE_DIRECTORIES(
 qt4_automoc(${corebindings_SRCS})
 
 
-add_library(corebindingsplugin SHARED ${corebindings_SRCS})
+add_library(corebindingsplugin MODULE ${corebindings_SRCS})
 target_link_libraries(corebindingsplugin ${KDE4_PLASMA_LIBS} ${QT_QTSCRIPT_LIBRARY} ${QT_QTDECLARATIVE_LIBRARY})
 
 install(TARGETS corebindingsplugin DESTINATION ${IMPORTS_INSTALL_DIR}/org/kde/plasma/core)
--- origsrc/kdebase-runtime-4.6.4/plasma/declarativeimports/graphicslayouts/CMakeLists.txt	2011-02-25 15:54:12.000000000 -0600
+++ src/kdebase-runtime-4.6.4/plasma/declarativeimports/graphicslayouts/CMakeLists.txt	2011-06-14 19:54:55.246936400 -0500
@@ -15,7 +15,7 @@ INCLUDE_DIRECTORIES(
 qt4_automoc(${graphicslayoutsbindings_SRCS})
 
 
-add_library(graphicslayoutsbindingsplugin SHARED ${graphicslayoutsbindings_SRCS})
+add_library(graphicslayoutsbindingsplugin MODULE ${graphicslayoutsbindings_SRCS})
 target_link_libraries(graphicslayoutsbindingsplugin ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTSCRIPT_LIBRARY} ${QT_QTDECLARATIVE_LIBRARY})
 
 install(TARGETS graphicslayoutsbindingsplugin DESTINATION ${IMPORTS_INSTALL_DIR}/org/kde/plasma/graphicslayouts)
--- origsrc/kdebase-runtime-4.6.4/plasma/declarativeimports/graphicswidgets/CMakeLists.txt	2011-02-25 15:54:12.000000000 -0600
+++ src/kdebase-runtime-4.6.4/plasma/declarativeimports/graphicswidgets/CMakeLists.txt	2011-06-14 19:55:29.114595400 -0500
@@ -14,7 +14,7 @@ INCLUDE_DIRECTORIES(
 qt4_automoc(${graphicswidgetsbindings_SRCS})
 
 
-add_library(graphicswidgetsbindingsplugin SHARED ${graphicswidgetsbindings_SRCS})
+add_library(graphicswidgetsbindingsplugin MODULE ${graphicswidgetsbindings_SRCS})
 target_link_libraries(graphicswidgetsbindingsplugin ${KDE4_PLASMA_LIBS} ${QT_QTSCRIPT_LIBRARY} ${QT_QTDECLARATIVE_LIBRARY})
 
 install(TARGETS graphicswidgetsbindingsplugin DESTINATION ${IMPORTS_INSTALL_DIR}/org/kde/plasma/graphicswidgets)
